src/classes/ADDR_Seasonal_BATCH.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>45.0</apiVersion>
+    <apiVersion>48.0</apiVersion>
```

Comment:
@m-kinkade  @skosgi1 @jstvz The maintenance of this seems very tedious, have we considered writing a task to inject the api version? I'm happy to work with you @skosgi1 and @m-kinkade  to demonstrate what I mean. (@jstvz  just tagging for your two cents.)

Reply:
@jofsky Sounds like a great idea!